package.json

```diff
@@ -46,7 +46,8 @@
     "alterschema": "^1.1.2",
     "change-case": "^4.1.2",
     "openapi-types": "9.3.0",
-    "typescript-json-schema": "^0.53.0"
+    "typescript-json-schema": "^0.53.0",
+    "winston": "^3.9.0"
```

Comment:
Why are you adding a new dependency?

Reply:
it's a logging library for node, i have removed it as it is not required.